Content.Shared/CCVar/CCVars.cs

```diff
@@ -840,6 +840,12 @@ public sealed class CCVars : CVars
         public static readonly CVarDef<bool> RestrictedNames =
             CVarDef.Create("ic.restricted_names", true, CVar.SERVER | CVar.REPLICATED);
 
+        /// <summary>
+        /// Allows flavor text (character descriptions)
+        /// </summary>
+        public static readonly CVarDef<bool> FlavorText =
+            CVarDef.Create("ic.flavor_text", true, CVar.SERVER | CVar.REPLICATED);
```

Comment:
we should default this to off IMHO. not sure we want this on LRP, when we get a MRP server we can turn it on